src/powerwrapper_papi/nrmpower_papi.c

```diff
@@ -277,12 +277,14 @@ int main(int argc, char **argv)
 	nrm_log_debug("%d NRM scopes initialized (%d NUMA and %d CPU)\n",
 	              n_scopes, n_numa_scopes, n_cpu_scopes);
 
-	long long *event_values;
+	long long *event_values, *event_totals;
 	nrm_time_t before_time, after_time;
 	int64_t elapsed_time;
 	double watts_value;
 
 	event_values = calloc(num_events, sizeof(long long));
+	event_totals = calloc(num_events, sizeof(double)); // converting then
+	                                                   // storing
```

Comment:
`event_totals` is an array of long long but is allocated as array of double, should be one or the other.